public/js/views/pay-methods.jsx

```diff
@@ -6,19 +6,30 @@ import CardList from 'components/card-list';
 import { gettext } from 'utils';
 
 
-export default class ManageCards extends Component {
+export default class PaymentMethods extends Component {
```

Comment:
I think it makes things easier when class names match file names as much as possible. Maybe `class PayMethods...` ?